torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -138,6 +144,12 @@ def is_public_torch_api(target):
     def is_safe_torch_function(target):
         """Allowlisted torch functions"""
         function_name = f"{target.__module__}.{target.__name__}"
+
+        if is_autograd_function(function_name):
```

Comment:
so existing hops have a `cacheable=True|False` boolean that you can flip, which I imagine is used by inductor's FXGraphCache. Can we just re-use the same infra here? (the main difference is just that `autograd.Function` is a HOP that exists in the dynamo graph, but "disappears" by the time we get to inductor)

Reply:
example: https://github.com/pytorch/pytorch/blob/main/torch/_higher_order_ops/triton_kernel_wrap.py#L773

Reply:
Oh yeah, we could do that as well, to make this code more generic. We'll still need to add FunctionCtx to the list of safe torch functions, though, since that isn't actually a higher order op